frame/mosaic/src/lib.rs

```diff
@@ -294,7 +305,8 @@ pub mod pallet {
 			amount: BalanceOf<T>,
 			swap_to_native: bool,
 			source_user_account: AccountIdOf<T>,
-			amm_swap_info: Option<AmmSwapInfo<NetworkIdOf<T>, RemoteAmmIdOf<T>>>,
+			amm_swap_info:
```

Comment:
not sure I see it is used anywhere. so adding is good, but make be make TODO or reference click up for later actual usage?